packages/react-vapor/src/components/radio/RadioSelect.tsx

```diff
@@ -70,8 +74,10 @@ export class RadioSelect extends React.PureComponent<IRadioSelectAllProps> {
     }
 
     private handleToggle(value: string, e: React.MouseEvent<HTMLElement>) {
-        this.props.onChange?.(value, this.props.id, e);
-        this.props.onChangeCallback?.(value, this.props.id, e);
+        if (!this.props.disabled) {
+            this.props.onChange?.(value, this.props.id, e);
+        }
+        this.props.onChangeCallback?.(value, this.props.id, e, this.props.disabled);

```

Comment:
Why is the callback called if the checkbox is disabled and not supposed to change?

Reply:
if you want to do something, its possible with that and you have your disabled props as parameter

Reply:
And I dont want to create a regression so I only disabled the onChange called by the dispatch